deploy/helm/sumologic/requirements.yaml

```diff
@@ -3,9 +3,9 @@ dependencies:
     version: 2.8.14
     repository: https://kubernetes-charts.storage.googleapis.com/
     condition: fluent-bit.enabled,sumologic.logs.enabled
-  - name: prometheus-operator
-    version: 8.13.8
-    repository: https://kubernetes-charts.storage.googleapis.com/
+  - name: kube-prometheus-stack
+    version: 9.3.4
+    repository: https://prometheus-community.github.io/helm-charts
     condition: prometheus-operator.enabled,sumologic.metrics.enabled
```

Comment:
Shouldn't we add an alias as well?

Reply:
you mean this?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/requirements.yaml#L9

Reply:
Alias was added in separate PR: #910